Form-Controls/index.html

```diff
@@ -6,22 +6,67 @@
     <title>My form exercise</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css">
   </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
+      <body>
+                <header>
+                  <h1>Product Pick</h1>
+                </header>
+                <form> 
+                      <div> <label for="full__name">Full name:</label>
+                            <input id="full-name" name="full_name" type="text" placeholder="Full Name" pattern=".{2,}" title="Please enter at least 2 characters" required>
+                      </div>
+                      <br>
+                      <div> 
+                            <label for="email">Email:</label>
+                            <input type="email" name="email" placeholder="Email"   pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
```

Comment:
Can you review your pattern? I was able to submit the form with an email like this:
user@company
Normally, a valid email would have a "dot" domain (user@company.com)
Further reading about email validation: https://www.abstractapi.com/guides/email-validation/email-address-format-validation